chunk_downloader.go

```diff
@@ -377,7 +400,16 @@ func decodeChunk(scd *snowflakeChunkDownloader, idx int, bufStream *bufio.Reader
 		if err != nil {
 			return err
 		}
+		var ctx context.Context
+		if isArrowRecordChanEnabled(scd.ctx) {
+			ch := make(chan []array.Record, 10) // TODO: is there a way to know how many array.Record will it be split up into within one routine?
```

Comment:
do we still need anything else than a channel with one element? since we send an array of records back

Reply:
yep forgot to change the 10 to 1, good catch thanks!